docs/core-concepts/core-sdk/cli/dotnet-run.md

```diff
@@ -7,55 +7,59 @@ dotnet-run -- Runs source code 'in-place' without any explicit compile or launch
 
 ## SYNOPSIS
 
-dotnet-run [--framework] [--configuration]
-    [--project] [--] [--help]
+`dotnet run [--framework] [--configuration]
+    [--project] [--help] [--]`
 
 ## DESCRIPTION
-The run command provides a convenient option to run your application from the  source code with one command. It compiles source code, generates an 
-output program and then runs that program. This command is useful for fast iterative development and can also be used 
-to run a source-distributed program (for example, website).
+The `dotnet run` command provides a convenient option to run your application from the source code with one command. 
+It compiles source code, generates an output program and then runs that program. 
+This command is useful for fast iterative development and can also be used to run a source-distributed program (for example, a website).
 
-This command relies on `dotnet-build(1)` to build source inputs to a .NET assembly, before launching the program. 
-The requirements for and handling of source inputs for this command are all inherited from the build command. 
+This command relies on [`dotnet build`](dotnet-build.md) to build source inputs to a .NET assembly, before launching the program. 
+The requirements for this command and the handling of source inputs are all inherited from the build command. 
 The documentation for the build command provides more information on those requirements.
 
 Output files are written to the child `bin` folder, which will be created if it doesn't exist. 
-Files will be overwritten as needed. Temporary files are written to the child `obj` folder.  
+Files will be overwritten as needed. 
+Temporary files are written to the child `obj` folder.  
 
-In case of a project with multiple specified frameworks, `dotnet run` will first select the .NET Core frameworks. If 
-those do not exist, it will error out. To specify other frameworks, use the `--framework` argument. 
+In case of a project with multiple specified frameworks, `dotnet run` will first select the .NET Core frameworks. If those do not exist, it will error out. To specify other frameworks, use the `--framework` argument.
+
+The `dotnet run` command must be used in the context of projects, not built assemblies. If you're trying to execute a DLL instead, you should use `dotnet` without any command like in the following example:
```

Comment:
@blackdwarf are you talking about linking to [overview.md](https://github.com/dotnet/core-docs/blob/master/docs/core-concepts/core-sdk/cli/overview.md) or [dotnet.md](https://github.com/dotnet/core-docs/blob/master/docs/core-concepts/core-sdk/cli/dotnet.md)?